pkg/chains/formats/all/all.go

```diff
@@ -16,6 +16,7 @@ package all
 
 import (
 	_ "github.com/tektoncd/chains/pkg/chains/formats/intotoite6"
+	_ "github.com/tektoncd/chains/pkg/chains/formats/intotoite6v2"
```

Comment:
I'm fine with either:
`slsa` branding is shorter while `intotoite6` (wouldn't mind changing it to just `intoto`) is a bit mouthful but more accurate here since `slsa` is a `predicate`, not `provenance`.

Reply:
I'm going to switch to `slsa` branding to close out this discussion 😁 . I'm happy to revert it later if we really want `intoto`.